### DISCUSSION TITLE
Moving TcMenu under a separate GitHub organization

### DISCUSSION THREAD

**Discussion by vzahradnik** [davetcc/tcMenu#356] [Ideas]
I was trying to move my tcmenu-python repository under your account but it failed. It's possible that we could figure it out but while I was thinking about it one thing came to my mind.

What if you created a new organization, e.g. TcMenu, and moved TcMenu related repositories there?

I can see at least two benefits:
- Users won't be confused with other repositories not related to the project, like your Joomla stuff
- You could add other contributors as organization members with various rights. I'd like to keep my access rights at least for the Python repository.

![image](https://github.com/davetcc/tcMenu/assets/4460766/80f311c7-935c-4fd9-be51-40984c00ff5a)

**Comment by davetcc** [comment]
I completely agree. I've accepted the transfer for now and made sure you still have commit rights, but after 4.0 is out I'll look into how this is possible.

**Reply by vzahradnik** [reply]
This sounds good to me as well. After releasing 4.0 we can get into this. And if you give me member rights, I could also play around with GitHub Actions and set up automated CI/CD. I have such setup already in place for my other Arduino projects. You can check here:

https://github.com/JSC-electronics/rfid-reader

(By the way, this library uses TaskManager for async operations.)
